### PR TITLE
DB-10424 Fix spotbugs for DB-10222 

### PR DESCRIPTION
**Pull request by arnaud-splice**

### CODE REVIEW

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1658/

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1659/

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1677/

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Review by martinrupp — APPROVED** [review]

File: db-drda/src/main/java/com/splicemachine/db/impl/drda/NetworkServerControlImpl.java [hunk 1]
```diff
@@ -1170,62 +1169,63 @@ public void checkShutdownPrivileges() throws SQLException {
 
         // the check
         try {
-            final Permission sp  = new SystemPermission(
+            //final Permission sp  =
```
Comment (yxia92) [inline db-drda/src/main/java/com/splicemachine/db/impl/drda/NetworkServerControlImpl.java:1172]: Please remove the commented out lines

**Review by yxia92 — APPROVED** [review]
+1

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**